{
  "paragraphs": [
    {
      "text": "import columnStoreExporter, time\nimport mysql.connector as mariadb\nfrom pyspark import SparkContext\nfrom pyspark.sql import Row, SQLContext\nfrom pyspark.sql.functions import rand, randn, sha1, sha2, md5\n\nurl \u003d \u0027jdbc:mysql://columnstore:3306\u0027\nproperties \u003d {\u0027user\u0027: \u0027root\u0027, \u0027driver\u0027: \u0027org.mariadb.jdbc.Driver\u0027}\n\nsc \u003d SparkContext(\"local\", \"MariaDB Spark ColumnStore Benchmark\")\nsqlContext \u003d SQLContext(sc)\n\n# SampleDataframe size parameter:\nasciiRange \u003d 128\nrandRange \u003d 1000\nhashRange \u003d 1000",
      "user": "anonymous",
      "dateUpdated": "2018-07-25 18:34:48.373",
      "config": {
        "colWidth": 12.0,
        "fontSize": 9.0,
        "enabled": true,
        "results": {},
        "editorSetting": {
          "language": "python",
          "editOnDblClick": false,
          "completionSupport": true
        },
        "editorMode": "ace/mode/python"
      },
      "settings": {
        "params": {},
        "forms": {}
      },
      "results": {
        "code": "ERROR",
        "msg": [
          {
            "type": "TEXT",
            "data": "Traceback (most recent call last):\n  File \"/tmp/zeppelin_python-4864983385112708688.py\", line 320, in \u003cmodule\u003e\n    raise Exception(traceback.format_exc())\nException: Traceback (most recent call last):\n  File \"/tmp/zeppelin_python-4864983385112708688.py\", line 308, in \u003cmodule\u003e\n    exec(code, _zcUserQueryNameSpace)\n  File \"\u003cstdin\u003e\", line 1, in \u003cmodule\u003e\n  File \"/usr/lib/python3/dist-packages/columnStoreExporter.py\", line 18, in \u003cmodule\u003e\n    import sys, pymcsapi, decimal, datetime\n  File \"/usr/lib/python3/dist-packages/pymcsapi.py\", line 17, in \u003cmodule\u003e\n    _pymcsapi \u003d swig_import_helper()\n  File \"/usr/lib/python3/dist-packages/pymcsapi.py\", line 16, in swig_import_helper\n    return importlib.import_module(\u0027_pymcsapi\u0027)\n  File \"/opt/conda/lib/python2.7/importlib/__init__.py\", line 37, in import_module\n    __import__(name)\nImportError: /usr/lib/python3/dist-packages/_pymcsapi.so: undefined symbol: PyBytes_AsStringAndSize\n\n"
          }
        ]
      },
      "apps": [],
      "jobName": "paragraph_1532543477508_-393310145",
      "id": "20180725-183117_161266874",
      "dateCreated": "2018-07-25 18:31:17.508",
      "dateStarted": "2018-07-25 18:34:48.427",
      "dateFinished": "2018-07-25 18:34:48.460",
      "status": "ERROR",
      "progressUpdateIntervalMs": 500
    },
    {
      "text": "try:\n    conn \u003d mariadb.connect(user\u003d\u0027root\u0027, host\u003d\u0027columnstore\u0027)\n    cursor \u003d conn.cursor()\n    cursor.execute(\"DROP DATABASE IF EXISTS benchmark\")\n    cursor.execute(\"CREATE DATABASE IF NOT EXISTS benchmark\")\n\nexcept mariadb.Error as err:\n    print(\"Error while preparing the database for the benchmark. %s\" %(err,))\n\nfinally:\n    if cursor: cursor.close()\n    if conn: conn.close()",
      "user": "anonymous",
      "dateUpdated": "2018-07-25 18:34:07.418",
      "config": {
        "colWidth": 12.0,
        "fontSize": 9.0,
        "enabled": true,
        "results": {},
        "editorSetting": {}
      },
      "settings": {
        "params": {},
        "forms": {}
      },
      "results": {
        "code": "ERROR",
        "msg": [
          {
            "type": "TEXT",
            "data": "Traceback (most recent call last):\n  File \"/tmp/zeppelin_python-4864983385112708688.py\", line 320, in \u003cmodule\u003e\n    raise Exception(traceback.format_exc())\nException: Traceback (most recent call last):\n  File \"/tmp/zeppelin_python-4864983385112708688.py\", line 313, in \u003cmodule\u003e\n    exec(code, _zcUserQueryNameSpace)\n  File \"\u003cstdin\u003e\", line 9, in \u003cmodule\u003e\nNameError: name \u0027cursor\u0027 is not defined\n\n"
          }
        ]
      },
      "apps": [],
      "jobName": "paragraph_1532543491593_112691222",
      "id": "20180725-183131_652527064",
      "dateCreated": "2018-07-25 18:31:31.593",
      "dateStarted": "2018-07-25 18:34:07.461",
      "dateFinished": "2018-07-25 18:34:07.479",
      "status": "ERROR",
      "progressUpdateIntervalMs": 500
    },
    {
      "user": "anonymous",
      "config": {},
      "settings": {
        "params": {},
        "forms": {}
      },
      "apps": [],
      "jobName": "paragraph_1532543541004_993689009",
      "id": "20180725-183221_205503746",
      "dateCreated": "2018-07-25 18:32:21.004",
      "status": "READY",
      "progressUpdateIntervalMs": 500
    }
  ],
  "name": "~Trash/Untitled Note 1",
  "id": "2DJQEASX3",
  "noteParams": {},
  "noteForms": {},
  "angularObjects": {
    "python:shared_process": [],
    "spark:shared_process": []
  },
  "config": {
    "isZeppelinNotebookCronEnable": false
  },
  "info": {}
}